itl/command-plugins.conf

```diff
@@ -549,6 +549,182 @@ object CheckCommand "http" {
 	vars.http_verbose = false
 }
 
+object CheckCommand "curl" {
+	import "ipv4-or-ipv6"
+
+	command = [ PluginDir + "/check_curl" ]
+
+	arguments += {
+		"-H" = {
+			description = "Host name argument for servers using host headers (virtual host). Append a port to include it in the header (eg: example.com:5000)"
```

Comment:
Move this and all remaining `description` attrs below the `value` and `set_if` attributes please.

Reply:
Why?

Reply:
> Why?

Just to be consistent with the other check commands.

Reply:
done